README.md

```diff
@@ -197,6 +197,17 @@ The following environment variables should be passed to the container in order t
 
 With the default parameters you can connect to JMX with `service:jmx:rmi://localhost:10002/jndi/rmi://localhost:10001/jmxrmi` without authentication.
 
+## Cluster
+
+The following environment variables should be passed to the container in order to configure the cluster configuration :
+
+|    VARIABLE              |  MANDATORY  |   DEFAULT VALUE          |  DESCRIPTION
+|--------------------------|-------------|--------------------------|----------------
+| EXO_CLUSTER | NO | false | Enable the cluster configuration
+| EXO_CLUSTER_NODE_NAME | NO | container id | Node name in the cluster
+| EXO_CLUSTER_HOSTS | NO | empty | Other nodes names
```

Comment:
Improve the description to explain the format to something like "Commas separated list of other nodes names of the eXo cluster"